iothub/device/src/Transport/HttpTransportHandler.cs

```diff
@@ -395,6 +396,33 @@ public override Task RejectAsync(string lockToken, CancellationToken cancellatio
                     cancellationToken), cancellationToken);
         }
 
+        internal Task<DirectMethodResult> InvokeMethodAsync(DirectMethodRequest directMethodRequest, Uri uri, CancellationToken cancellationToken)
+        {
+            TimeSpan timeout = GetInvokeDeviceMethodOperationTimeout(directMethodRequest);
+            var customHeaders = new Dictionary<string, string>
+            {
+                { CustomHeaderConstants.ModuleId, $"{this.deviceId}/{this.moduleId}" }
```

Comment:
Should never happen, but wonder if we should check that moduleId exists here.. (for future proofing).